src/tablib/formats/_ods.py

```diff
@@ -110,22 +110,33 @@ def import_sheet(cls, dset, sheet, headers=True, skip_lines=0):
 
         dset.title = sheet.getAttribute('name')
 
-        def is_real_cell(cell):
-            return cell.hasChildNodes() or not cell.getAttribute('numbercolumnsrepeated')
+        def repeat(cell):
+            try:
+                n = int(cell.getAttribute("numbercolumnsrepeated"))
+            except Exception:
```

Comment:
Can we use more specific exceptions here?

Reply:
It might be possible just to catch `ValueError` and `AttributeError`, but I really do mean to catch all errors, so I can't recommend it. I think it's safe to assume that if anything goes wrong here, the cell value should not be repeated.